### PR TITLE
Bureaucracy Overhaul

### PR DESCRIPTION
**Pull request by Crushtoe**
...of the jobs page, which was half-done and poorly written.

### CODE REVIEW

**Comment by JJRcop** [comment]
would a "clickbait" label be too much?

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -372,7 +372,7 @@
...
...ers<BR>3. Distribute Supplies<BR><BR><B>Cargo Technician</B><BR>Cargo Technicians operate Cargo's supply requests, move objects on and off the shuttle, and distribute goods personally.<BR>1. Approve & Deny Supply Requests<BR>2. Load & Unload the Cargo Shuttle<BR>3. Distribute Supplies<BR><BR><B>Shaft Miner</B><BR>Shaft Miners mine and refine resources such as plasma, diamonds, and iron for the station's use in robotics, chemistry, and research.<BR>1. Mine Resources<BR>2. Refine Resources<BR><BR><B>Janitor</B><BR>Janitors are in charge of keeping the station clean and appearing well maintained.<BR>1. Clean the Station<BR>2. Dispose of garbage<BR><BR><B>Bartender</B><BR>The Bartender is in charge of providing the crew with alcoholic & non-alcoholic beverages. When people need to drown their sorrows, they turn to him.<BR>1. Give Drink to the Thirsty<BR>2. Keep Peace in the Bar<BR><BR><B>Cook</B><BR>The Cook is in charge of preparing fine meals to feed the Station's crew.<BR>1. Give Food to the Hungry<BR><BR><B>Botanist</B><BR>The Botanist is in responsible for the growth of plants for the Cook to use in meal preparation and for raw consumption by the populous.<BR>1. Grow Food<BR><BR>"
```
Comment (xxalpha) [inline code/modules/paperwork/paper.dm]: Divide this into multiple lines with \ so it's readable.

Comment (tkdrg) [inline code/modules/paperwork/paper.dm]: - You missed adding a `<br>` right after `<b>Security Officer</b>`.
- Punctuation is inconsistent in the numbered lists. Some end with a period, but most don't. Make it all or nothing.
- Atmos techs have "They keep the station hot, cold, and breathable." which sounds weird and contradicts itself.
- The geneticist one has a rogue period after the /etc in the description.
- The Botanist one has a misspelling: `populace` not `populous`.

**Comment by KorPhaeron** [comment]
@Crushtoe are you going to finish this?

**Comment by KorPhaeron** [comment]
Let me know if you want this re-opened/are going to finish

**Pull request closed without merging**